From 8fb9ad34706ed223b65685eb52c52a399c9933e9 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 2 May 2014 14:37:19 -0400
Subject: [misc] mei: fix compilation error with missing WATCHDOG_CORE

Message-id: <1399041470-21330-47-git-send-email-prarit@redhat.com>
Patchwork-id: 79333
O-Subject: [RHEL7.1 PATCH BZ 1088953 46/77] mei: fix compilation error with missing WATCHDOG_CORE
Bugzilla: 1088953
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088953

commit 208008c322fd1ffdd07e7fc95910154174ce0ff3
Author: Tomas Winkler <tomas.winkler@intel.com>
Date:   Mon Feb 10 14:33:53 2014 +0200

    mei: fix compilation error with missing WATCHDOG_CORE

    Kconfig is not transitive so INTEL_ME_TXE has to depend
    on WATCHDOG_CORE as well

    ERROR: "watchdog_unregister_device" [drivers/misc/mei/mei.ko] undefined!
    ERROR: "watchdog_register_device" [drivers/misc/mei/mei.ko] undefined!

    Reported-by: Fengguang Wu <fengguang.wu@intel.com>
    Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/mei/Kconfig b/drivers/misc/mei/Kconfig
index 5881679..d23384d 100644
--- a/drivers/misc/mei/Kconfig
+++ b/drivers/misc/mei/Kconfig
@@ -37,7 +37,7 @@ config INTEL_MEI_ME
 config INTEL_MEI_TXE
  tristate "Intel Trusted Execution Environment with ME Interface"
  select INTEL_MEI
- depends on X86 && PCI
+ depends on X86 && PCI && WATCHDOG_CORE
  help
    MEI Support for Trusted Execution Environment device on Intel SoCs
 
-- 
1.7.1